tests/reftests/inplace.test

```diff
@@ -134,7 +134,7 @@ The following actions will be performed:
   - install qux     u
 
 <><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
--> retrieved ongoing.dev  (git+file://${BASEDIR}/ongoing#master)
+-> retrieved ongoing.dev  (no changes)
```

Comment:
is this change in the test showing that `git reset --hard HEAD` is called when using `--inplace`?

Reply:
no, it is this one https://github.com/ocaml/opam/pull/4879/files#diff-702f2fec9e39e8a7d4cc0180c593309659368bd0a5a7cc500e2dbacda1e1f6d8R92

Reply:
to be sure it is not called there, could we add:
```
### : Checking that calling opam install --inplace does not overwrite the changes made to local files
### <ongoing/ongoing.txt>
not versionned
```
before this last test, and
```
### cat ongoing/ongoing.txt
not versionned
```
after.

Reply:
I'm not sure to understand, `ongoing` already uses a unversioned file in its build, `newfile.txt`. You want to add new unversioned modifications in the repo to be sure that the last ones are not dropped?

Reply:
no no, `ongoing.txt` is versionned. I want to check that its content stays the same and isn’t reverted to its versionned content (which is `versionned` instead of `not versionned` as overwritten above)

Reply:
the new version no more changes this test